tests/Common/FilesTest.php

```diff
@@ -174,7 +181,7 @@ public function testFileUpload($filePath, FileUploadOptions $options)
         // check attachment, download
         $client = new Client();
         $response = $client->get($file['url']);
-        $this->assertStringStartsWith('attachment;', (string) $response->getHeader('Content-Disposition')[0]);
+        $this->assertStringStartsWith('attachment', (string) $response->getHeader('Content-Disposition')[0]);
```

Comment:
tuhle hlavičku `attachment;` se středníkem se mi nepodařilo nastavit v Azure. Domnívám se, že to není až tak důležitý a tak jsem tomu nevěnoval větší úsili a nechal jsem tam `attachment` bez středníku. Když tak to ještě zkusím upravit.

Reply:
On tam tenhle problém vlastně nezvnikne, protože to id je v názvu kontejneru a soubor se ukládá pod normálním názvem. Nebo tobě se to někde stáhlo jinak ?

Reply:
vznikne, to jméno `name` se posílá úplně bokem a do toho jména v S3 se vůbec nepřenáší.

Reply:
- mám souboru "file.csv"
 - vytvořím si pro něj místo na ABS (`file/prepare ... name=file.csv`)
 - soubor tam nahraju do `...microsoft.com/exp-180-123/file.csv`
 - dám si vygenerovat url pro stažení a stáhne se mi soubor, který se chce uložit jako `file.csv`

Měl jsem vzato, že tenhle problém je jenom u S3, kdy před názem souboru vygeneruje id, které je pak součástí souboru, ale při stažení ty nechceš, aby se ti to stáhlo s tím id. 

Nebo mi něco nedochází ?

Reply:
jo ale soubor se netvytváří jenom přes API pomocí prepare. šlo např. hlavně o soubory generované jako soubory tabulek https://github.com/keboola/connection/blob/master/legacy-app/application/modules/storage/jobs/TableExport.php#L228 . K tomu se dostaneme tak to zatím asi neřeš teda. Možná se to tam vyřeší stejně.

Reply:
Aha, tak až na to narazím, tak to fixnu jestli to bude potřeba.